src/net/sourceforge/kolmafia/textui/command/SkeletonCommand.java

```diff
@@ -22,11 +21,11 @@ public SkeletonCommand() {
 
   public static final Object[][] SKELETONS =
       new Object[][] {
-        {"warrior", IntegerPool.get(WARRIOR)},
-        {"cleric", IntegerPool.get(CLERIC)},
-        {"wizard", IntegerPool.get(WIZARD)},
-        {"rogue", IntegerPool.get(ROGUE)},
-        {"buddy", IntegerPool.get(BUDDY)},
+        {"warrior", WARRIOR},
```

Comment:
Ya know, these scores of Object[][] are really calling to be Enums or something. Some other PR. :)